gbpservice/neutron/nfp/configurator/api/v1/controllers/controller.py

```diff
@@ -0,0 +1,184 @@
+import json
+import subprocess
```

Comment:
Should this file name be controller.py? Can't this be called as configurator_controller.py? In future, if we add new controller files, controller.py name will not be meaningful.